docker-compose.yml

```diff
@@ -98,7 +98,7 @@ services:
 
       PHP_OPCACHE_VALIDATE_TIMESTAMPS: 1
       PHP_IDE_CONFIG: serverName=lpa-front-app
-      # ENABLE_XDEBUG: 'true'
+      ENABLE_XDEBUG: 'true'
```

Comment:
Did you definitely intend to check in this change?  If I recall correctly, while it'd be convenient and logical to have xdebug always turned on in dev, it was commented out because it can be horribly slow to start up, so we only want to use it when we have to debug. Not an ideal situation cos then one has to keep turning it back on. I wonder if we could use an environment variable for it actually, that would sort the issue of sometimes needing it and sometimes not?

Reply:
Nope, I keep forgetting about that.